src/middlewares/sendEmail.js

```diff
@@ -15,9 +15,9 @@ const transporter = nodemailer.createTransport({
 
 let link;
 if (process.env.NODE_ENV === 'development') {
-  link = 'http://localhost:3000/verification/';
+  link = 'http://localhost:3000/api/v1/user/verification/';
```

Comment:
for the seck of better and clear implementation , we can retrieve the frontend and backend url from .env file